<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mask_manager</class>
 <widget class="QDialog" name="mask_manager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>254</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Mask Management</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0" rowspan="5">
    <widget class="QTableWidget" name="masks_table">
     <property name="contextMenuPolicy">
      <enum>Qt::CustomContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::DoubleClicked</set>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="sortingEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>true</bool>
     </attribute>
     <column>
      <property name="text">
       <string>Name</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Visible</string>
      </property>
      <property name="textAlignment">
       <set>AlignCenter</set>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Remove</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="import_masks">
     <property name="text">
      <string>Import Masks</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="export_masks">
     <property name="text">
      <string>Export Masks</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="panel_buffer">
     <property name="text">
      <string>Masks to Panel Buffer</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QPushButton" name="view_masks">
     <property name="text">
      <string>View Masks</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="hide_masks">
     <property name="text">
      <string>Hide all masks</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>masks_table</tabstop>
  <tabstop>panel_buffer</tabstop>
  <tabstop>import_masks</tabstop>
  <tabstop>export_masks</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
